cdap-ui/app/directives/log-viewer/log-viewer.js

```diff
@@ -14,182 +14,166 @@
  * the License.
  */
 
-angular.module(PKG.name + '.commons')
-  .directive('myLogViewer', function ($filter, $timeout, $state, $location) {
-
-    var capitalize = $filter('caskCapitalizeFilter'),
-        filterFilter = $filter('filter');
+function LogViewerController () {
+  'ngInject';
+
+  this.isMessageExpanded = false;
+  this.isTimeDisplayed = true;
+  this.isLevelDisplayed = true;
+  this.isSourceDisplayed = true;
+  this.isMessageDisplayed = true;
```

Comment:
this should a directive configuration, pass these as a scope isolate, and default everything to true, unless user specify a column to false. Also it's much better to keep these as a single object.